.github/workflows/chromatic.yml

```diff
@@ -85,7 +85,7 @@ jobs:
         # This action must be manually updated to the latest commit from
```

Comment:
This is no longer true. Dependabot will handle updates

Reply:
removed via f10ef46d5a360a77abe4c0c02ee15b6c2e2df3c1